src/microbe_stage/PatchManager.cs

```diff
@@ -156,60 +152,40 @@ private void HandleCellSpawns(Patch patch)
         foreach (var entry in patch.SpeciesInPatch)
         {
             var species = entry.Key;
+            var populationHere = entry.Value;
```

Comment:
Two concerns on this:
a) wouldn't it be more efficient not to instantiate variables here
b) if we stick by this, we probably should settle for a solution when dealing with dictionary entries, cause other parts in the code do not create such variables.

And, if clarity is the issue, perhaps a more telling name than entry could be considered.

Reply:
I've recently discovered that you can even destructure variables (usually) with something like: `foreach(var (thing, secondThing) in container)` which is very awesome. I would expect the compiler to optimize away any temporary variables or at least that isn't much of a problem if this is code that isn't ran every frame.